Exercise_4/docs/part-1.md

```diff
@@ -0,0 +1,330 @@
+
+??? abstract "Test Order Code Structure and Examples"
+    Test Order codes are 5 digit numeric codes appended with a dash and 6th number. Form: XXXXX-X
+
+    e.g.,
+        As an example, one of the more popular codes is: `2160-0` for `Creatinine [Mass/volume] in Serum or Plasma`
+        Source:  [https://loinc.org/2160-0/](https://loinc.org/2160-0/)
+
+    Additional attributes associated with LOINC test codes are:
+
+    * Test ID
+    * Test Order Name
+    * Order LOINC Code
+    * LOINC Attributes
+    * Method name
+
+
+    e.g.,
+    An example can be found here: [https://loinc.org/2160-0/](https://loinc.org/2160-0/) and [https://loinc.org/kb/users-guide/introduction/](https://loinc.org/kb/users-guide/introduction/)
+
+
+    | Test ID | Test Order Name | Order LOINC Code | LOINC Attributes | Method Name |
+    |---------|-----------------|------------------|------------------|-------------|
+    | 10AIH  | Factor X Inhibitor Profile, Professional Interpretation | 69049-5 | Coagulation specialist review:Imp:Pt:XXX:Nar: | Only orderable as a reflex. For more information see 10INE / Factor X Inhibitor Evaluation, Plasma. Medical Interpretation |
+
+??? abstract "Test Result Code Structure and Examples"
+    LOINC Test Result codes are also 5 digit numeric codes appended with a dash and 6th number. Form: XXXXX-X
+
+    Additional attributes associated with LOINC Test Result codes are:
+
+    * LOINC Values Updated
+    * Test ID
+    * Test Order Name
+    * Result ID
+    * Result LOINC Code
+    * LOINC Attributes
+    * Method Name
+
+    | LOINC Values Updated | Test ID | Test Order Name | Result ID | Test Result Name| Result LOINC Code | LOINC Attributes | Method Name |
+    |----------------------|---------|-----------------|-----------|-----------------|-------------------|------------------|-------------|
+    | 2022-03-11 | CNSA | Central Nervous System Consultation, Autopsy, Varies | 71070 | Interpretation | 18743-5 | Autopsy report:Find:Pt:{Setting}:Doc:{Role} | Includes hermatoxylin and eosin stains. Additional special stains or procedures necessary for diagnosis will be billed separately. |
+
+!!! check "For more information on LOINC codes"
+    Additonal infomration on the **LOINC coding terminology** is available here: [https://loinc.org]( https://loinc.org)
+
+??? question "Knowledge Check: Where would I look to find details on the specific LOINC code?"
+    The `Method name` attribute provides additional details on the specific LOINC code.
+
+### CPT
+
+**Current Procedural Terminology (CPT)** codes are numbers assigned to each task and service a healthcare provider offers. They include medical, surgical, and diagnostic services. Insurers use CPT codes to determine how much money to pay a provider for a given task or service.
+
+!!! Note "Code Similarities"
+    CPT codes are also a subset of codes comprising the Healthcare Common Procedure Coding System (HCPCS) used for Medicare reimbursements.
```

Comment:
Source? This information seems irrelevant since HCPCS is never discussed otherwise.

Reply:
HCPCS is important, and therefore CPT is important, because this is how providers bill Medicare.